.travis.yml

```diff
@@ -9,7 +9,7 @@ services:
 
 env:
   global:
-    - DOCKER_IMAGE=adshares/adserver:devops
+    - DOCKER_IMAGE=awlodarkiewicz/devops:adserver
```

Comment:
What is wrong with `adshares/adserver:devops`?
We can create `adshares/devops:adserver`.